datev_export/models/res_company.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2023 initOS GmbH
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    datev_consultant_number = fields.Char(
+        string="Consultant Number",
+        size=32,
+        help="Number from 1000 to 9999999",
+    )
+
+    datev_client_number = fields.Char(
+        string="Client Number",
+        size=32,
+        help="Number from 0 to 99999",
```

Comment:
same here

Reply:
I limitted the size a bit further. Wouldn't it be better to use an Integer here or atleast a regex constraint? I don't find anything official talking about the bounds of these numbers. What do you think?